packages/project/src/shared/tree-utils/find-closest-non-inline-parent.ts

```diff
@@ -7,7 +7,13 @@ export const findClosestNonInlineParent = (
 ): Instance | undefined => {
   const path = getInstancePath(rootInstance, instanceId);
   path.reverse();
-  return path.find(
-    (item) => getComponentMeta(item.component).isInlineOnly === false
-  );
+  return path.find((item) => {
+    const type = getComponentMeta(item.component).type;
+    return (
+      type === "container" ||
+      type === "control" ||
+      type === "embed" ||
+      type === "rich-text"
```

Comment:
same here

Reply:
I think Oleg is right. We call `findClosestNonInlineParent` under this condition:

```
if (getComponentMeta(instance.component).type === "rich-text-child")
```

I think it would be good to have the opposite condition inside the function. Otherwise we might get some confusing behaviour when a new type is added if we forget to add it here.